lib/hypervisor/hv_kvm/monitor.py

```diff
@@ -500,6 +502,7 @@ def HotAddNic(self, nic, devid, tapfds=None, vhostfds=None, features=None):
         fdnames.append(fdname)
 
       arguments.update({
+        "romfile": "",
```

Comment:
Did you discover, that an empty romfile is needed, too, when `vhost_net` is enabled?

Reply:
uh, thanks for spotting. This is actually wrong here ('romfile' is only relevant to the device_add structure, not to netdev_add)